github-repos/src/redux/actions.js

```diff
@@ -1,11 +1,11 @@
-
-
-export const fetchRepos = username => (dispatch) => {
-  dispatch({ type: 'LOADING' });
+export const fetchRepos = username => dispatch => {
+  dispatch({ type: "LOADING" });
   fetch(`https://api.github.com/users/${username}/repos`)
-    .then((response) => {
-      dispatch({ type: 'SUCCESS' });
+    .then(response => {
+      dispatch({ type: "SUCCESS" });
       return response.json();
     })
-    .then(data => dispatch({ type: 'FETCH_REPOS', payload: data.map(repo => repo.name) }));
```

Comment:
It is very likely that someone can mispell the action type, prefer to set a constant for it in a constants file like actionTypes.